specs/merge/client_settings.md

```diff
@@ -0,0 +1,17 @@
+# The Merge -- Client Settings
+
+**Notice**: This document is a work-in-progress for researchers and implementers.
+
+This document specifies configurable settings that merge clients are expected to ship with.
+
+### Override terminal total difficulty
+
+To coordinate changes to [`terminal_total_difficulty`](fork-choice.md#transitionstore), clients
+should have a setting `--terminal-total-difficulty-override`.
+
+If `TransitionStore` has already been initialized, this just changes the value of
```

Comment:
I would also add that the `engine_terminalTotalDifficultyOverride` message should be sent to the execution client.

Reply:
I might instead specify it wrt the EIP 3675 events rather than the API. This leaves it generic enough for alternative merge-client implementations (e.g. a unified client that doesn't  utilize an API for comms on the layer

Reply:
```suggestion
If `TransitionStore` has already [been initialized](./fork.md#initializing-transition-store), this alters the previously initialized value of
```

Reply:
> I might instead specify it wrt the EIP 3675 events rather than the API. This leaves it generic enough for alternative merge-client implementations (e.g. a unified client that doesn't utilize an API for comms on the layer

agree with this